src/common/intl.cpp

```diff
@@ -1898,7 +1898,8 @@ wxString GetDateFormatFromLangInfo(wxLocaleInfo index)
 wxString wxLocale::GetInfo(wxLocaleInfo index, wxLocaleCategory cat)
 {
 // TODO: as of 2014 Android doesn't has complete locale support (use java api)
-#if !(defined(__WXQT__) && defined(__ANDROID__))
+// DONE: Supported by NDK21d
+//#if !(defined(__WXQT__) && defined(__ANDROID__))
```

Comment:
Could you please remove these comments completely rather than leaving them if they're not relevant any longer?

Reply:
Agreed. Will fix.